packages/harpguru-core/src/components/option/option.tsx

```diff
@@ -115,23 +108,27 @@ export const Option = (props: OptionProps): React.ReactElement => {
             ]}
           >
             <OptionValue
-              id={activeOptionId}
+              id={visibleOptionList[2]}
+              isActive={true}
               setFunction={setFunction}
               style={styles.activeOptionValue}
             />
           </Animated.View>
           <OptionValue
-            id={inactiveOptionId3}
+            id={visibleOptionList[3]}
+            isActive={false}
             setFunction={setFunction}
             style={styles.nextOptionValue}
           />
           <OptionValue
-            id={inactiveOptionId4}
+            id={visibleOptionList[4]}
+            isActive={false}
             setFunction={setFunction}
             style={styles.distantOptionValue}
           />
           <OptionValue
-            id={inactiveOptionId5}
+            id={visibleOptionList[5]}
+            isActive={false}
             setFunction={setFunction}
             style={styles.distantOptionValue}
```

Comment:
Yea I'm pretty sure these can just determine their own styles now based on the `isActive` prop

Reply:
Done